kickoff/views/releases.py

```diff
@@ -30,8 +28,8 @@ def cmpReleases(x, y):
                 return 1
         elif y.complete:
             return -1
-        return cmp(y._submittedAt, x._submittedAt)
-    return sorted(getReleases(ready = ready), cmp=cmpReleases)
+        return y._submittedAt < x._submittedAt
```

Comment:
I'm not sure if this is a valid replacement of `cmp()` - the new variant returns `True` or `False` vs old `-1/0/1`